topics/dev/tutorials/core-contributing/contributing.md

```diff
@@ -0,0 +1,18 @@
+To contribute to galaxy, a GitHub account is required. Changes are proposed via a [pull request](https://docs.github.com/en/github/collaborating-with-pull-requests). This allows the project maintainers to review the changes and suggest improvements.
```

Comment:
This looks like a good snippet, I'm going to move it into the snippet/faq framework so other tutorials can easily take advantage of it as well. (Since we now have a couple that discuss contributing to galaxy)